test/topicFilters.spec.ts

```diff
@@ -61,6 +83,57 @@ async function afterEachTest(): Promise<void> {
   await namespace.close();
 }
 
+async function sendOrders(): Promise<void> {
+  const data = [
+    { Color: "blue", Quantity: 5, Priority: "low" },
+    { Color: "red", Quantity: 10, Priority: "high" },
+    { Color: "yellow", Quantity: 5, Priority: "low" },
+    { Color: "blue", Quantity: 10, Priority: "low" },
+    { Color: "blue", Quantity: 5, Priority: "high" },
+    { Color: "blue", Quantity: 10, Priority: "low" },
+    { Color: "red", Quantity: 5, Priority: "low" },
+    { Color: "red", Quantity: 10, Priority: "low" },
+    { Color: "red", Quantity: 5, Priority: "low" },
+    { Color: "yellow", Quantity: 10, Priority: "high" },
+    { Color: "yellow", Quantity: 5, Priority: "low" },
+    { Color: "yellow", Quantity: 10, Priority: "low" }
+  ];
```

Comment:
Lets move this out of this function, so that we can then use `data.length` instead of the hard-coded number 12 in the tests